components/well/stories/template.js

```diff
@@ -14,25 +13,13 @@ export const Template = ({
   return html`
     <span class=${classMap({
       [rootClass]: true,
-      [`${rootClass}--size${size?.toUpperCase()}`]: typeof size !== "undefined",
       ...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
     })}>
       <em>Well done is better than well said.</em>
       <br>
       <strong>Benjamin Franklin</strong>
       <br><br>
       Well said Ben!
```

Comment:
Can you pass the content in as a variable so that we can embed this story in others as needed? We've been using the `content` variable name and pass the data in as an array. Something like this here:

```suggestion
      ${content}
```

and this in the stories.js:

```
content: [
  html`<em>Well done is better than well said.</em><br><strong>Benjamin Franklin</strong><br><br>Well said Ben!`
]

Reply:
makes sense, thanks for the example!